### ISSUE TITLE
pip 2.0.9 install was failing

### ISSUE THREAD

**Issue by jgoppert** [ArduPilot/pymavlink#38]
I took down the 2.0.9 package as it won't build with pip. 2.0.8 works.

**Comment by khancyr** [comment]
@jgoppert hi, Could you detail the error ?

**Comment by jgoppert** [comment]
Started out that it couldn't find future. After I tried to fix that (once by using master and once by doing it myself), the generated python files weren't being included. I tried to fix it, but gave up for now.

**Comment by khancyr** [comment]
Yes, new dependencies are future, lxml and python-dev .
Normally dependencies should have been solved in master but no new version has been made and it has been update on mavlink repo.

**Comment by jgoppert** [comment]
@khancyr I tried updating it using master. All packaging attempts failed (2.0.10, 2.0.11, 2.0.12). I gave up at that point. The last issue I ran into was that the include path for mavnative needed to be absolute since pip was not building the extension in source. Probably best to bump the next version to 2.0.13 when you get it working. Pypi was updating slowly so I didn't realize my 2.0.9.post0 actually worked as well. So if you are stuck in a packaging nightmare like I was maybe just start doing 2.0.13.post0, 2.0.13.post1 etc. until you get it working.
